pkgs/tools/text/num-utils/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchurl, perl }:
+stdenv.mkDerivation rec {
+  pname = "num-utils";
+  version = "0.5";
+
+  src = fetchurl {
+    url = "http://suso.suso.org/programs/num-utils/downloads/num-utils-${version}.tar.gz";
+    sha256 = "0kn6yskjww2agcqvas5l2xp55mp4njdxqkdicchlji3qzih2fn83";
+  };
+
+  buildInputs = [ perl ];
+
+  patchPhase = ''
+    sed -i -e 's|-o 0 -g 0||' Makefile
+  '';
+  makeFlags = [ "TOPDIR=\${out}" "PERL=${perl}/bin/perl" ];
```

```suggestion
  makeFlags = [ "TOPDIR=${placeholder "out"}" "PERL=${perl}/bin/perl" ];
```

Is that a runtime or compile time perl?

---

It's both; the Makefile bakes the path into the installed scripts.

---

That will not work when cross compiling. We need to split buildTime and runtime perl.

---

True, okay, I removed the compile time usage with another patch to the Makefile. Now it's only a runtime dep.